docs/setup-ci.md

```diff
@@ -0,0 +1,104 @@
+---
+id: setup-ci
+title: Setup Continuous Integration Pipeline for an RNW App
+---
+
+This guide will help you get started on setting up your very first continuous integration pipeline for a React Native for Windows app.
+
+## Setting Up a Continuous Integration Pipeline using Github Actions
+
+When done developing your app, it's good practice to setup a CI pipeline with automated build and tests to avoid any future regressions. There are many services available for setting up a CI pipeline. We'll use [GitHub Actions](https://docs.github.com/en/actions/getting-started-with-github-actions/about-github-actions) as an example here since it doesn't require any extra account setup if you are already hosting your code on GitHub, also the default VM image has all the tools we needed pre-installed.
```

Comment:
```suggestion
When done developing your app, it's good practice to setup a CI pipeline with automated build and tests to avoid any future regressions. There are many services available for setting up a CI pipeline. We'll use [GitHub Actions](https://docs.github.com/actions/getting-started-with-github-actions/about-github-actions) as an example here since it doesn't require any extra account setup if you are already hosting your code on GitHub, also the default VM image has all the tools we needed pre-installed.
```